app/views/articles/_actions.html.erb

```diff
@@ -1,20 +1,22 @@
 <div class="article-actions" id="article-reaction-actions">
-  <button id="reaction-butt-like" class="article-reaction-butt like-reaction-button" data-category="like" alt="heart-emoji" title="heart">
-    <img alt="heart" src="<%= asset_path("emoji/emoji-one-heart.png") %>" /><span class="reaction-number" id="reaction-number-like"></span>
-  </button>
-  <button id="reaction-butt-unicorn" class="article-reaction-butt unicorn-reaction-button" data-category="unicorn" alt="unicorn-emoji" title="unicorn">
-    <img alt="unicorn" src="<%= asset_path("emoji/emoji-one-unicorn.png") %>" /><span class="reaction-number" id="reaction-number-unicorn"></span>
-  </button>
-  <button id="reaction-butt-readinglist" class="article-reaction-butt readinglist-reaction-button" data-category="readinglist" alt="unicorn-emoji" title="reading list">
-    <img alt="reading list" src="<%= asset_path("emoji/emoji-one-bookmark.png") %>" /><span class="reaction-number" id="reaction-number-readinglist"></span>
-  </button>
-  <a class="article-actions-tweet-button"
-     id="article-actions-tweet-button"
-     rel="noopener"
-     target='_blank'
-     href='https://twitter.com/intent/tweet?text="<%= u @article.title.strip %>" by <%= @article.user.twitter_username ? "@" + @article.user.twitter_username : @article.user.name %> %23DEVcommunity https://dev.to<%= @article.path %>'>
-    <img src="<%= asset_path("twitter.svg") %>" alt="twitter logo">
-  </a>
+  <% if @article.published? %>
```

Comment:
Now that I think about it more, if a user has a published post that they decide to unpublish in the future, they won't see the previous reactions anymore. I think because of this scenario we'll want to keep the reactions sidebar there. Or you could change the boolean to:
```
if @article.published? || @article.positive_reactions_count.positive?
```

Reply:
@Zhao-Andy in this case I think it is easier to hide the reactions rather than showing them and worrying about disabling them.